### PR TITLE
Demo blaze sign up first steps

### PR DESCRIPTION
**Pull request by kevinsmba**

### CODE REVIEW

File: DemoBlaze Testing/mainSelectorsDemoBlaze.js
Comment (jquinterot) [inline DemoBlaze Testing/mainSelectorsDemoBlaze.js:1]: You can call it DemoBlazePage, due to is the normal standars for page object pattern

File: mainTest.js
Comment (jquinterot) [inline mainTest.js:1]: There is a git word at the end of the assertion, please remove it

File: package.json
Comment (jquinterot) [inline package.json:1]: I think that here  you only need to change the js file name from mainTest to mainSaignUp

File: DemoBlaze Testing/mainSelectorsDemoBlaze.js
Comment (kevinsmba) [reply DemoBlaze Testing/mainSelectorsDemoBlaze.js:1]: are you meaning to change the file mainSelectorsDemoBlaze.js to DemoBlazePage.js?

Comment (jquinterot) [reply DemoBlaze Testing/mainSelectorsDemoBlaze.js:1]: It should be both in order to be aligned

File: package.json
Comment (kevinsmba) [reply package.json:1]: for some reason that I still don't understand is only allowing me to have the full route of the document given that is in a folder, is not working as you stated.

**Review by jquinterot — APPROVED** [review]

**Pull request merged**